src/cesium_ion/main.rb

```diff
@@ -0,0 +1,448 @@
+require 'json'
+require 'net/http'
+require 'rbconfig'
+require 'socket'
+require 'uri'
+
+require 'aws-sdk-s3'
+require 'zip'
+
+module Cesium::IonExporter
+
+  @@debug = false
+  @@local = false
+
+  @@clientId = 14
+
+  @@baseServer = 'https://cesium.com/'
+  @@apiServer = 'https://api.cesium.com/'
+  if @@local
+    @@baseServer = 'http://composer.test:8081/'
+    @@apiServer = 'http://api.composer.test:8081/'
+  end
+
+  @@callbackHost = 'localhost'
+  @@callbackPort = 10101
+  @@callbackDomain = "#{@@callbackHost}:#{@@callbackPort}"
+  @@callbackPath = '/oauth'
+  @@callbackUri = "http://#{@@callbackDomain}#{@@callbackPath}"
+
+  @@modelExportOptions = {
+    :triangulated_faces => true,
+    :doublesided_faces => true,
+    :edges => false,
```

Comment:
What are `edges`?

Reply:
```
Export Edges: When this option is selected, the exported file maintains the appearance of a SketchUp model with visible edges. Note that stand-alone edges are always exported.
```
So I believe its the non-photorealistic black edges that sketchup draws.